inginious/frontend/static/css/themes/Makefile

```diff
@@ -0,0 +1,16 @@
+BOOTSWATCH_URL	= https://github.com/thomaspark/bootswatch.git
+BASE_THEME	= yeti
+
+compile:
+	if [ ! -d "bootswatch" ]; then make install; fi
```

Comment:
It would be more convenient if all these steps could be carried out by grunt to avoid adding development dependencies. In case the yeti sources are provided in our repo the build configuration file should be provided too and can be modified to place the file in the appropriate locations.